pvlib/pvsystem.py

```diff
@@ -693,12 +695,12 @@ def sapm(module, poa_direct, poa_diffuse, temp_cell, airmass_absolute, aoi):
         (1 + module['Aimp']*(temp_cell - T0)) )
 
     dfout['v_oc'] = (( module['Voco'] +
-        module['#Series']*delta*np.log(Ee) + Bvoco*(temp_cell - T0) )
+        module['Cells in Series']*delta*np.log(Ee) + Bvoco*(temp_cell - T0) )
```

Comment:
The spaces are only removed from the index field, not the column names. I was actually tempted to change the column names to make them lowercase, so that they would be slightly more robust to Sandia's tweaks. I could make them lower and remove spaces, seems reasonable.